src/SlateAsInputEditor/index.js

```diff
@@ -190,13 +235,13 @@ const SlateAsInputEditor = React.forwardRef((props, ref) => {
       case 'block_quote':
         return <blockquote {...attributes}>{children}</blockquote>;
       case 'code_block':
-        return <pre {...attributes}>{children}</pre>;
```

Comment:
I don't understand this. Shouldn't we be letting Slate control the `contentEditable`?